### DISCUSSION TITLE
When changing the title with meta plugin two title tags appears

### DISCUSSION THREAD

**Discussion by Hashen110** [quasarframework/quasar#12350] [CLI - SSR mode] [answered]
I want to change the title on every page using the quasar meta plugin but if I forget to change the title on a page I want to use the default title that's in index.template.html file.

It seems when putting the title tag in index.template.html and using the quasar meta plugin, two title tags will appear inside the head tag instead of updating the current title tag.

The app is running in SSR mode.

```html
<!DOCTYPE html>
<html>
  <head>
    <title><%= productName %></title>
    <meta name="description" content="<%= productDescription %>">
    <meta property="og:site_name" content="<%= productName %>">
    <meta property="og:description" content="<%= productDescription %>">
  </head>
</html>
```

```js
setup() {
  useMeta({
    title: 'Title',
    meta: {
      ogTitle: {
      property: 'og:title',
      content: 'Title',
    },
  },
});
```

**Comment by rstoenescu** [comment, marked as answer]
Either use Meta plugin or put a title in src/index.template.html
Not both ways simultaneously. Meta has optimizations under the hood, but does not account for non-Meta generated tags.
